src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -178,9 +189,11 @@ public void park(boolean reversed) {
         stop();
         if(reversed){
             //not using calculateSetpoint because these are less than one full rotation
-            turningPIDController.setReference(Math.PI/4, ControlType.kPosition);  
+            int errorCode = turningPIDController.setReference(Math.PI/4, ControlType.kPosition).value;  
+            if(errorCode != 0){DriverStation.reportError("An Error has occured in SwerveModule.park() for SparkMaxPIDController.setReference() Code:" + REVLibError.fromInt(errorCode).toString(), null);}
```

Comment:
Which swerve module failed?  You have the ID

Reply:
done, ID added to all error messages